wireapp/wire-ios-mono
# feat: indicate client with valid certificate [WPB-3220]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-3220" title="WPB-3220" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />WPB-3220</a>  [iOS] 5.3.14 Indicate client with valid certificate
  </td></table>
  <br />
 
































----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Solutions

Changes the layout for displaying Proteus and MLS information on Client list screen when navigating from the Conversation screen. Also displays Proteus verification and E2E Identity status images

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test
Steps
1. Go to any conversation
2. Tap on other user
3. Tap on Devices Tab
4. Relevant IDs and images should be shown based on if it is a proteus protocol or MLS protocol.

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)
![IMG_0006](https://github.com/wireapp/wire-ios/assets/17357739/32b36a10-c260-48cd-a7e7-aa3c320feb73)

![IMG_0007](https://github.com/wireapp/wire-ios/assets/17357739/bf63896c-7dbe-4b05-ab93-de5bad9cc83e)

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
